Makefile

```diff
@@ -126,6 +131,12 @@ watch: ## automatically compiles sources when changed
 	bin/watch src "$(MAKE) compile"
 .PHONY: watch
 
+# This rule requires to install the inotify-tools dependency (Linux Only)
+# https://github.com/inotify-tools/inotify-tools/wiki
+watch-plugins: ## automatically copies build plugin sources when changed
+	bin/watch src/plugins "$(MAKE) plugins"
```

Comment:
Running two watchers on the same files creates a race condition :disappointed: 
(`watch-plugins` triggers while `yarn watch` is still running)